scripts/sql/32603301_reference-5-1-0.up.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO "public"."chart_ref" ("location", "version","deployment_strategy_path", "is_default", "active", "created_on", "created_by", "updated_on", "updated_by") VALUES
```

Comment:
While adding the new chart , add the name column as well.

**name='Rollout Deployment'**